packages/experimental-app-router/src/server/routeHandler/tokenHandler.ts

```diff
@@ -19,7 +19,7 @@ export async function tokenHandler(req: Request) {
     }
 
     const { url } = req;
-    const code = new URL(url).searchParams.get('code');
+    const code = new URL(url).searchParams.get('code') ?? undefined;
```

Comment:
`searchParams.get()` returns `null` if there isn't a search param. In current Faust, we return `undefined`, so this change brings parity to the existing API router. The authorize endpoint in the plugin technically supports both:

https://github.com/wpengine/faustjs/blob/1e50244db87185b03aee3851dc95e052174d862a/plugins/faustwp/includes/rest/callbacks.php#L167-L172

So I figure there is no need to make a changeset of this since it doesn't affect users.